_format/1.1/index.md

```diff
@@ -1232,6 +1232,65 @@ Accept: application/vnd.api+json
 A server **MUST** return `403 Forbidden` in response to an unsupported request
 to create a resource with a client-generated ID.
 
+#### <a href="#crud-sideposting" id="crud-sideposting" class="headerlink"></a> Sideposting
+
+A server **MAY** accept a request to create related resources along with the primary
+resource, which is referred to as *sideposting*. A related resource **MUST**
+be placed within the `included` member, and have a `temp-id` member the value of which
+is an arbitrary string that uniquely identifies a related resource within the request
+document. The related resources **MUST** be referred to (from the linkage data of
+resource relationships) via a *temporary resource identifier object* (that is an
+object with a single `temp-id` member referring to a related resource).
+
+The document **MUST** still respect the full linkage requirement.
+
+For instance, a new article might be created along with two tags with the following 
+request:
+
+```http
+POST /articles HTTP/1.1
+Content-Type: application/vnd.api+json
+Accept: application/vnd.api+json
+
+{
+  "data": {
+    "type": "articles",
+    "attributes": {
+      "title": "Sideposting with json:api"
+    },
+    "relationships": {
+      "tags": {
+        "data": [{ "type": "tags", "id": "9" },
+                 { "temp-id": "1" },
```

Comment:
I imagined the situation where we are accepting user generated IDs. Client generates unique id and send it as `temp-id`. But for example if client generates UUIDv5, this means that IDs could intersect between entities of different types. Possibility critically low, but it exists.